python/surf/protocols/ssi/_SsiPrbsRateGen.py

```diff
@@ -160,3 +214,7 @@ def __init__(self, **kwargs):
             pollInterval = 1,
             mode         = "RO",
         ))
+
+    @staticmethod
+    def convMbps(var):
+        return var.dependencies[0].value() * 8e-6
```

Comment:
self.convMbps is still being used but not linked to the new clock_freq parameter

Reply:
It's not related to clock frequency.
It converts bytes/sec to Mbits/sec.